hypertrace-gradle-docker-java-application-plugin/src/main/resources/application-start-script.template.sh

```diff
@@ -23,10 +23,9 @@
 
 set -e
 
-CLASSPATH="/app/resources:/app/classes:/app/localLibs/*:/app/orgLibs/*:/app/externalLibs/*"
-if [ "\$FIPS_ENABLED" = "true" ]; then
-  CLASSPATH="/usr/share/java/bc-fips/*:\${CLASSPATH}"
-  echo "Adding the additional FIPS libs to the classpath"
-fi
-
+# Remove trailing colon if present
+FIPS_CLASSPATH="\${FIPS_CLASSPATH%:}"
```

Comment:
I know there's some differents in `sh` vs `bash` for the string manipulation capabilities - have we verified these are all compatible with `sh` since that's the shell here?